src/components/Person/Person.jsx

```diff
@@ -1 +1,7 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person_name">{person.name};</h2>
+    <p className="Person_age">{person.age};</p>
+    <p className="Person_partner">{person.partner};</p>
```

Comment:
The `Person` component should handle the logic for displaying the partner's information based on the person's marital status and gender. You need to implement this logic to check if the person is married and then display the partner's name accordingly. This aligns with the task requirement: "If the person is married, show their partner's name and use the correct word (husband/wife) depending on the person's sex."